app/models/assignment.js

```diff
@@ -41,9 +41,9 @@ export default class AssignmentModel extends Model {
 
   // if the each dcpPublichearinglocation and dcpDateofpublichearing properties are filled in dispositions array,
   // then hearings have been submitted for that project
-  @computed('dispositions.@each.{dcpPublichearinglocation,dcpDateofpublichearing}')
+  @computed('dispositionsByRole.@each.{dcpPublichearinglocation,dcpDateofpublichearing}')
   get hearingsSubmitted() {
-    const dispositions = this.get('dispositions');
+    const dispositions = this.get('dispositionsByRole');
```

Comment:
I know this was from a previous PR, but I'm confused by why we need the `dispositionsByRole` computed. Shouldn't the zap-api already deliver an assignments with its attached dispositions filtered to `assignment.dcp_Lupteammemberrole`?

Reply:
@godfreyyeung it should but it's not and I don't know why. I think @hannahkates was looking at it one point. I want to make this explicit check so we're not submitting dispositions against the wrong roles.

Reply:
There's also no actual explicit join table between a lup team and disposition/contact. It's inferred through these fields.